bcftbx/cmdparse.py

```diff
@@ -56,12 +57,12 @@ class CommandParser:
     Create a simple CommandParser using:
 
     >>> p = CommandParser()
-    
+
     Add a 'setup' command:
 
     >>> p.add_command('setup',usage='%prog setup OPTIONS ARGS')
 
-    Add options to the 'setup' command using normal OptionParser methods,
+    Add options to the 'setup' command using normal ArgumentParser methods,
```

Comment:
Update usage documentation to indicate that either `OptionParser` (the default) or `ArgumentParser` can be used as a subparser.